gamemodes/terrortown/entities/weapons/weapon_ttt_m16.lua

```diff
@@ -83,7 +87,12 @@ end
 ---
 -- @ignore
 function SWEP:Reload()
-	if self:Clip1() == self.Primary.ClipSize or self:GetOwner():GetAmmoCount(self.Primary.Ammo) <= 0 then return end
+	if
+		self:Clip1() == self.Primary.ClipSize
+		or self:GetOwner():GetAmmoCount(self.Primary.Ammo) <= 0
+	then
+		return
+	end
```

Comment:
I think I know why this looks weird to me. The third line here starts with an operator (`or`). The first line as well (`if`). But the second line looks kinda lost without an operator. This is why I prefer
```
if ThisIsTheFirstCheck()
	or ThisIsTheSecondCheck()
	and ThisIsTheThirdCheck()
then
	-- do some stuff
and
```